src/pages/iou/request/MoneyTemporaryForRefactorRequestParticipantsSelector.js

```diff
@@ -79,14 +81,25 @@ function MoneyTemporaryForRefactorRequestParticipantsSelector({
 }) {
     const {translate} = useLocalize();
     const styles = useThemeStyles();
-    const [searchTerm, setSearchTerm] = useState('');
+    const [searchTerm, debouncedSearchTerm, setSearchTerm] = useDebouncedState('');
+    const [didScreenTransitionEnd, setDidScreenTransitionEnd] = useState(false);
     const {isOffline} = useNetwork();
     const personalDetails = usePersonalDetails();
 
     const offlineMessage = isOffline ? `${translate('common.youAppearToBeOffline')} ${translate('search.resultsAreLimited')}` : '';
 
     const maxParticipantsReached = participants.length === CONST.REPORT.MAXIMUM_PARTICIPANTS;
 
+    const isOptionsDataReady = useMemo(() => ReportUtils.isReportDataReady() && OptionsListUtils.isPersonalDetailsReady(personalDetails), [personalDetails]);
+
+    useEffect(() => {
+        const interactionTask = InteractionManager.runAfterInteractions(() => {
+            setDidScreenTransitionEnd(true);
+        });
+
+        return interactionTask.cancel;
+    }, []);
```

Comment:
Hey @lukemorawski can you help me understand why we need to check for the transition state?